src/modules/logger/logged_topics.cpp

```diff
@@ -68,6 +68,7 @@ void LoggedTopics::add_default_topics()
 	add_optional_topic("heater_status");
 	add_topic("home_position");
 	add_topic("hover_thrust_estimate", 100);
+	add_topic("humidity_temperature");
```

Comment:
I would add an interval limit as a precaution. 2 Hz sufficient for humidity and temperature?

```suggestion
	add_topic("humidity_temperature", 500);
```

Reply:
Yes, 2 Hz is enough